_posts/2016-01-06-little-big-blue.md

```diff
@@ -0,0 +1,26 @@
+---
+layout: post
+title: Little Big Blue
+date: 2016-01-06 15:45:45 +05:30
+homepage: https://github.com/ninapetrop/Little-Big-Blue
+download: https://github.com/ninapetrop/Little-Big-Blue/archive/master.zip
+demo: # unavailable
+author: Nina Petropoulos
+thumbnail: little-big-blue.png
+license: MIT License
+license_link: https://github.com/ninapetrop/Little-Big-Blue/blob/master/license.txt
+___
```

Comment:
Hello and thanks for the PR! There is a typo in here, it should be `---` not `___`. Please fix it :smile: